src/managers/GuildChannelManager.js

```diff
@@ -21,6 +21,19 @@ class GuildChannelManager extends BaseManager {
     this.guild = guild;
   }
 
+  /**
+   * The number of channels in this managers cache excluding thread channels
+   * that do not count towards a guild's maximum channels restriction.
+   * @type {number}
+   * @readonly
+   */
+  get channelCountWithoutThreads() {
+    return this.cache.reduce((acc, channel) => {
+      if (ThreadChannelTypes.includes(channel.type)) return acc;
+      return acc++;
```

Comment:
Won't this always return with 1 less? 
```suggestion
      return ++acc;
```

Reply:
Actually, that'd always return 0, come think of it.

Reply:
Yeah, always returns 0 🤦